zha/zigbee/cluster_handlers/manufacturerspecific.py

```diff
@@ -120,7 +120,7 @@ def __init__(self, cluster: zigpy.zcl.Cluster, endpoint: Endpoint) -> None:
 class OppleRemoteClusterHandler(ClusterHandler):
     """Opple cluster handler."""
 
-    REPORT_CONFIG = ()
+    REPORT_CONFIG: tuple[AttrReportConfig, ...] = ()
```

Comment:
In a future PR, we can remove empty `REPORT_CONFIG`s for all `ClusterHandler`s, as this is already the default.
Otherwise, we can just specify the type everywhere. It was only needed here for now, as mypy otherwise complained about an impossible equality check in the added test.